### PR TITLE
Feature/multi countries

### PR DESCRIPTION
**Pull request by BilelJegham**
Add : 
 - Multi-select countries
 - Add switch format xAxis (date <-> day number )

### CODE REVIEW

**Comment by BilelJegham** [comment]
@RemiTheWarrior I think it's OK, I let you check.

https://epidemic-simulator-git-feature-multi-countries.bileljegham.now.sh/#/

**Comment by RemDelaporteMathurin** [comment]
Is there a way to order the legend by country ?
image

**Comment by RemDelaporteMathurin** [comment]
image
Concerning the default behaviour, I can't see the series deaths - France (no simu).

**Comment by RemDelaporteMathurin** [comment]
image
Is it possible to ensure the default display (infected + deaths, sim + actual data) ?

Way to reproduce:
- Set France
- Set UK
- click on "display count date"

**Comment by BilelJegham** [comment]
I think it's related to issue #8 : problem of cache browser 

Try with `ctrl+F5`.

**Comment by RemDelaporteMathurin** [comment]
okay then, what about sorting the legend by country ?

**Comment by RemDelaporteMathurin** [comment]
image
Is it possible to keep points for actual data ?

**Comment by RemDelaporteMathurin** [comment]
image
It still seems to have some bugs when comparing countries with "display count date"

File: src/views/AccueilPage.vue [hunk 1]
```diff
@@ -1,35 +1,37 @@
...
       <md-card-header>
-        <div class="md-title">{{this.country}}</div>
+        <div class="md-title">{{this.countriesSelect.join(', ')}}</div>
       </md-card-header>
       <md-card-content>
-        <ChartH :series="this.seriesAll"/>
-        <p class="text-center"><small>Timezone : UTC+1</small></p>
+        <ChartH v-if="this.number" :series="this.seriesAllNumber" :number="true"/>
+        <ChartH  v-else :series="this.seriesAll" />
+         <p class="text-center"><md-switch v-model="number" class="md-primary">Compare country</md-switch><br/>
```
Comment (RemDelaporteMathurin) [inline src/views/AccueilPage.vue]: `Compare country` should be `Compare countries`

**Review by RemDelaporteMathurin — CHANGES_REQUESTED** [review]

**Pull request merged**